telemetry/glean/relay-server-metrics.yaml

```diff
@@ -0,0 +1,159 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# For more information on configuring this file:
+# https://mozilla.github.io/glean/book/reference/yaml/metrics.html
+
+# This file is also used to generate privaterelay/glean/server_events.py
+# which is used to generate the Glean pings-as-logs.  See that file for the
+# re-generation command.
+
+# Status: Draft
+# This work is tracked in https://mozilla-hub.atlassian.net/browse/MPP-3352
+#
+# This is the first Relay integration with Glean, and is focused on the
+# data needed to determine if a Relay user is engaged with the service,
+# creating masks and getting emails. There are more metrics planned in the
+# future, after we get over the technical hurdles of the first integration.
+#
+# Remaining critical path work in the code:
+# - Request a data review, update bugs and data_reviews
+# - Make required changes from data review
+# - Delete this status section
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+email_mask:
+  deleted:
+    type: event
+    description: A Relay user deletes an email mask.
+    expires: never
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1882565
+      - https://mozilla-hub.atlassian.net/browse/MPP-3352
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1882565
+    notification_emails:
+      - jwhitlock@mozilla.com
```

Comment:
suggestion (non-blocking): `relay-team@mozilla.com` ?

Reply:
The Glean documents have no guidance, so I reached out to internal Slack channel. According to @perrymcmanis144, we have to have a single person for metrics that do not expire. We could add a mailing list, but not replace the single person email.

However, `relay-team@mozilla.com` doesn't look like a good choice to me. There are many people on that list, including people no longer on the project. Once I have some experience with the kinds of emails sent to this address, we could add a maintained list with the people who should get notices and can do something about them.

Reply:
Sorry for the drive-by, I just wanted to add some additional context to what Perry shared with you.

The Glean docs lack guidance here because this is a Data Stewarding concern (see the [wiki on Data Collection](https://wiki.mozilla.org/Data_Collection) for more info).  That being said, I'll file a bug in Glean to try and add some breadcrumbs to this information to make it easier to understand the expectations here.

Permanent data-collection policy from Trust & Legal currently requires an IC to be responsible for the collection, in order to audit it from time to time to determine that it is still useful and needed. Temporary collections that expire don't run the same risk of indefinitely collecting some data that might not be useful or problematic.

As a side-note, many teams/project create a specific group email to serve as a backup for an IC bearing this responsibility (should the IC leave or not be available, etc.).